src/FurCoNZ.Web/Services/ReminderService.cs

```diff
@@ -0,0 +1,153 @@
+            foreach (var order in expiredOrdersSinceLastSend)
+            {
+                // Gather metadata
+                var toAddresses = new MailAddressCollection
+                {
+                    new MailAddress (order.Email, order.Name),
+                };
+                var subject = $"Order #{order.Id} has expired";
+
+                // Prepare template
+                var model = new OrderExpiredNotificationViewModel();
+                var message = await _viewRenderService.RenderToStringAsync("EmailTemplate/ThirtyDayReminder", model, cancellationToken: cancellationToken);
+
+                // Send message
+                await _emailService.SendEmailAsync(toAddresses, subject, message, cancellationToken: cancellationToken);
+            }
+
+            // Update last run time so we don't resend messages
+            // TODO: Track individual reminders against orders so we don't resend all if one message in a batch is faulty
+            lastSend.LastRun = currentRun;
+
+            await _dbContext.SaveChangesAsync(cancellationToken);
+        }
+
+        public async Task SendCancelledOrdersAsync(CancellationToken cancellationToken = default)
+        {
+            // Get the last time the reminders were sent
+            var lastSend = await _dbContext.RemindersLastRuns.FindAsync(new[] { "SendCancelledOrders" }, cancellationToken);
+            if (lastSend == null)
+            {
+                lastSend = new Models.RemindersLastRun
+                {
+                    ReminderService = "SendCancelledOrders",
+                    LastRun = DateTimeOffset.MinValue,
+                };
+                _dbContext.RemindersLastRuns.Add(lastSend);
+            }
+            var lastRun = lastSend.LastRun;
+            var currentRun = DateTimeOffset.Now;
+
+            // Get the orders that we will send reminders for
+            var expiredOrdersSinceLastSend = await _dbContext.Orders.AsNoTracking()
+                .Where(o =>
+                    !o.Audits.Any() && // No payments have been made
+                    o.CreatedAt.AddDays(_options.UnpaidOrderExpiryDays) <= currentRun && // Order has expired
+                    o.CreatedAt.AddDays(_options.UnpaidOrderExpiryDays) >= lastRun // Reminder has not been sent
+                )
+                .Select(o => new
+                {
+                    o.Id,
+                    o.CreatedAt,
+                    o.OrderedBy.Name,
+                    o.OrderedBy.Email
+                })
+                .ToListAsync(cancellationToken);
+
+            // Send emails for each order
+            foreach (var order in expiredOrdersSinceLastSend)
+            {
+                // Gather metadata
+                var toAddresses = new MailAddressCollection
+                {
+                    new MailAddress (order.Email, order.Name),
+                };
+                var subject = $"Order #{order.Id} has expired";
```

Comment:
Could the View's `ViewData["Title"]` be used here to prevent use of fixed strings?

Reply:
Awesome idea. Logged as #81, will tackle this after release.